core/src/main/java/com/sap/ai/sdk/core/AiCoreService.java

```diff
@@ -160,6 +161,7 @@ public AiCoreDeployment forDeploymentByScenario(@Nonnull final String scenarioId
   @Nonnull
   protected Destination getBaseDestination()
       throws DestinationAccessException, DestinationNotFoundException {
+    Dotenv.configure().ignoreIfMissing().load();
```

Comment:
(Question)

Does it really work?
`Dotenv.configure().ignoreIfMissing().load();` really changes the behavior of `System.getenv`?

Reply:
(Major)

![image](https://github.com/user-attachments/assets/5282e4ea-4920-44a2-8910-199662389ff1)

Reply:
Fixed, I used Dotenv only and it can catch both env vars and the env file.
**The env var has priority over the `.env` file.**

Reply:
I added it in the readme